<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="image">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>500</height>
         </size>
        </property>
        <property name="text">
         <string>Image</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="vertical_spacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="image_num_label">
        <property name="text">
         <string>Image 0 of 0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="circumference_label">
        <property name="text">
         <string>Circumference:
50.123456789 cm</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="export_button">
        <property name="statusTip">
         <string>Export data.</string>
        </property>
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="slice_settings_label">
        <property name="text">
         <string>Slice Settings</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="slice_settings_text">
        <property name="statusTip">
         <string>Slice settings selected from main window.</string>
        </property>
        <property name="text">
         <string>X rotation: 123째
Y rotation: 45째
Z rotation: 67째
Slice: 89째</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="adjust_slice_button">
        <property name="statusTip">
         <string>Go back to main window to adjust slice settings.</string>
        </property>
        <property name="text">
         <string>Adjust</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="trace_settings_label">
        <property name="text">
         <string>Trace Settings</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="trace_settings_text">
        <property name="text">
         <string>Threshold: 123
Blurring: 45
Another setting: 67
Another setting: 89</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="adjust_trace_button">
        <property name="statusTip">
         <string>Adjust trace settings.</string>
        </property>
        <property name="text">
         <string>Adjust</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1071</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_add_image"/>
    <addaction name="action_remove_image"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_documentation"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_open">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Select file(s) to be opened in .nrrd, .nii.gz, or .nii.gz formats.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_documentation">
   <property name="text">
    <string>Documentation</string>
   </property>
   <property name="statusTip">
    <string>Link to documentation page.</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit.</string>
   </property>
   <property name="shortcut">
    <string>Meta+W, Ctrl+W</string>
   </property>
  </action>
  <action name="action_add_image">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add Image</string>
   </property>
   <property name="statusTip">
    <string>Add a single image.</string>
   </property>
  </action>
  <action name="action_remove_image">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Remove Image</string>
   </property>
   <property name="statusTip">
    <string>Remove a single image.</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
